python/processors/coin_flip/README.md

```diff
@@ -0,0 +1,371 @@
+
+- An API key for the GRPC backend. If you do not have one, you should use your API gateway to obtain one [here](https://github.com/aptos-labs/aptos-indexer-processors)
+- The [Aptos CLI](https://aptos.dev/tools/aptos-cli/)
+- [Python](https://www.python.org/downloads/) and [Poetry](https://python-poetry.org/docs/#installing-with-the-official-installer)
+- This cloned repository on your local machine `https://github.com/aptos-labs/aptos-indexer-processors`
+
+We use postgresql as our database in this tutorial. You're free to use whatever you want, but this tutorial is geared towards postgresql for the sake of simplicity. We use the following database configuration and tools:
+- [Postgresql](https://www.postgresql.org/download/)
+    - We will use a database hosted on `localhost` on the port `5432`, which should be the default
+    - When you create your username, keep track of it and the password you use for it
+    - You can view a tutorial for installing postgresql and psql [here](https://www.digitalocean.com/community/tutorials/how-to-install-postgresql-on-ubuntu-22-04-quickstart) tool to set up your database more quickly
+    - If you want to easily view your database data, consider using a GUI like [DBeaver](https://dbeaver.io/) or [pgAdmin](https://www.pgadmin.org/)
+
+Explaining how to create a database is beyond the scope of this tutorial, so if you don't know how to do it, you could check out tutorials on how to create a database with the `psql` tool.
+
+## Setup your environment
+
+### Setup the postgresql database
+
+Make sure to start the `postgresql` service:
+
+The command for Linux/WSL might be something like:
+
+```shell
+sudo service postgresql start
+```
+
+For mac, if you're using brew, start it up with:
+
+```shell
+brew services start postgresql
+```
+
+Create your database with the name `coin_flip`, where our username is `user` and our password is `password`.
+
+If your database is set up correctly, and you have the `psql` tool, you should be able to run the command `psql -d coin_flip`.
+
+### Setup your local environment with poetry and grpc
+
+If you haven't yet, make sure to read the main indexer [README guide](https://github.com/aptos-labs/aptos-indexer-processors).
+
+You can also check out the python-specific broad overview of how to create an indexer processor [here](https://github.com/aptos-labs/aptos-indexer-processors/tree/main/python).
+
+At the very least, make sure to install these tools and setup your poetry environment:
+
+```shell
+pip install grpcio-tools
+poetry install
+
+python3 -m grpc_tools.protoc --proto_path=./proto --python_out=python --pyi_out=python --grpc_python_out=python  \
+          proto/aptos/bigquery_schema/v1/transaction.proto  \
+          proto/aptos/indexer/v1/raw_data.proto \
+          proto/aptos/internal/fullnode/v1/fullnode_data.proto \
+          proto/aptos/transaction/v1/transaction.proto \
+          proto/aptos/util/timestamp/timestamp.proto
+```
```

Comment:
@banool is it easy to add the protobuf types and the autogenerated files into a python package? That way the user only has to call `poetry install` to get started.